centml/cli/cluster.py

```diff
@@ -158,17 +174,19 @@ def get(type, id):
         )
 
 
-# Define common deployment
+# load hardware pricing choices
+def load_hw_choices():
+    hw_to_id_map, _ = get_hw_to_id_map()
+    return list(hw_to_id_map.keys())
+
+
+# Define common deployment options
 def common_options(func):
     hw_to_id_map, _ = get_hw_to_id_map()
```

Comment:
you need to remove this line or else the unauthenticated call is still made

Reply:
yeah actually its still giving issue, yeah i have also cross checked may be i need to remove the decorator mechanism, currently debugging the same

Reply:
removed the decorator logic and moved the creation of inference and compute deployment flow to use direct click related commands